controls/def.cf

```diff
@@ -93,30 +93,34 @@ bundle common def
         if => not( isvariable( "control_server_allowusers_policy_server" ) );
 
     policy_server::
-      "control_server_allowusers" -> { "CFE-3544" }
+      "control_server_allowusers" -> { "CFE-3544", "ENT-6666" }
         handle => "def_control_server_allowusers_policy_server",
         slist => { @(control_server_allowusers_policy_server) },
+        meta => { "inventory", "attribute_name=cf-runagent allowed users" },
```

Comment:
```suggestion
        meta => { "inventory", "attribute_name=Allowed users for cf-runagent" },
```

Reply:
I don't have a problem with your suggestions, the only reason I started with cf-runagent is so that they would be tightly grouped with the alphabetical sort.